src/ontology/mondo.Makefile

```diff
@@ -268,8 +269,45 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
-	
+#############################################
+##### One-time scripts ######################
+#############################################
+# MedGen conflicts (Aug 2023) pipeline
+# - https://github.com/monarch-initiative/mondo-ingest/issues/273
+.PHONY: address-medgen-conflicts-aug2023
+address-medgen-conflicts-aug202x3: address-medgen-conflicts-aug2023-deletes address-medgen-conflicts-aug2023-adds
+
+# - MedGen conflicts: dependencies
+tmp/July2023_CUIReports_FromMedGentoMondo.xlsx:
+	mkdir -p tmp && wget "https://github.com/monarch-initiative/mondo-ingest/files/12029712/July2023_CUIReports_FromMedGentoMondo.xlsx" -O $@
+
+# - MedGen conflicts: analysis
+tmp/mondo-edit.obo.tmp.diff: mondo-edit.obo.tmp
+	-diff mondo-edit.obo mondo-edit.obo.tmp > $@
+
+tmp/report-qc-medgen-conflicts-update-diff.tsv: tmp/mondo-edit.obo.tmp.diff
+	python ../scripts/medgen_conflicts_removals_diff_analysis.py -i tmp/mondo-edit.obo.tmp.diff -o $@
+
+tmp/report-qc-medgen-conflicts-adds-diff.tsv: $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv
+	python ../scripts/medgen_conflicts_add_xrefs_diff_analysis.py -c $(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv -i $(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv -o $@
+
+# - MedGen conflicts: deletes
+.PHONY: address-medgen-conflicts-aug2023-deletes
+address-medgen-conflicts-aug2023-deletes: mondo-edit.obo.tmp
+	mv mondo-edit.obo.tmp mondo-edit.obo
+
+mondo-edit.obo.tmp: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx mondo-edit.obo
+	python ../scripts/medgen_conflicts_removals.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -I mondo-edit.obo -o $@
+
+# - MedGen conflicts: adds
+.PHONY: address-medgen-conflicts-aug2023-adds
+address-medgen-conflicts-aug2023-adds: tmp/report-qc-medgen-conflicts-adds-diff.tsv
+
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromConflictResolution.tsv: tmp/July2023_CUIReports_FromMedGentoMondo.xlsx
+	python ../scripts/medgen_conflicts_add_xrefs.py -i tmp/July2023_CUIReports_FromMedGentoMondo.xlsx -o $@
 
+$(TEMPLATES_DIR)/ROBOT_addMedGen_fromIngest.tsv:
```

Comment:
## [Robot template: UID sources #13](https://github.com/monarch-initiative/medgen/pull/13#pullrequestreview-1618699434)
Updated MedGen ingest repo with [this PR](https://github.com/monarch-initiative/medgen/pull/13#pullrequestreview-1618699434), which, for the robot template it is generating, added a `source_id` column so that for any UIDs we are ingesting, we include the source UMLS CUI they are mapped to.

Uploaded the new file to [releases](https://github.com/monarch-initiative/medgen/releases/tag/2023-09-03), then ran `make ../templates/ROBOT_addMedGen_fromIngest.tsv -B` to update the template in this repository.

Reply:
I have removed all these MEDGEN mappings for now - this PR is now focussing on the UMLS conflicts